packages/plugin-github/tsup.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+    entry: ["src/index.ts"],
+    outDir: "dist",
+    sourcemap: true,
+    clean: true,
+    format: ["esm"], // Ensure you're targeting CommonJS
+    external: [
+        "dotenv", // Externalize dotenv to prevent bundling
+        "fs", // Externalize fs to use Node.js built-in module
+        "path", // Externalize other built-ins if necessary
+        "@reflink/reflink",
+        "@node-llama-cpp",
+        "https",
+        "http",
+        "agentkeepalive",
+        "fs/promises",
+        "csv-writer",
+        "csv-parse/sync",
+        "path",
+        "url",
```

Comment:
I added these because it was causing issues on runtime between common vs module so feel free to remove these unless you are using them thanks!